networking/network_observability/network-observability-overview.adoc

```diff
@@ -24,7 +24,9 @@ The Network Observability Operator requires the following Operators:
 [id="network-observability-operator"]
 == Network Observability Operator
 
-The Network Observability Operator deploys a monitoring pipeline to collect and enrich the network flows. The eBPF agent, which is deployed as a `daemonset` object, creates the network flows. The network flows are then collected and enriched with the Kubernetes metadata, before storing in Loki.
+The Network Observability Operator deploys a monitoring pipeline to collect and enrich the network flows. The eBPF agent, which is deployed as a `daemonset` object, creates the network flows. The network flows are then collected and enriched with the Kubernetes metadata before storing in Loki. 
+
+The Flow Collector resource is created during the installation of the Operator and enables configuration of network flow collection.  When the Flow Collector is deployed, new dashboards are available in the {product-title} console. 
```

Comment:
Don't confuse the CRD (Flow Collector API) with the resource or instance (also called CR or custom resource).  After installation of Operator, it just provides the Flow Collector API.  Again, when the flow collector resource is created, that's when things are deployed.